main.py

```diff
@@ -168,20 +191,25 @@ async def submit(
         "Elixir",
     ],
 ):
+    check_submission_channel()
     await interaction.response.defer()
     submission = await handle_submission(interaction, attachment, language)
 
-    response_message = f"Thanks for uploading, {interaction.user.display_name}! Recieved {language} file: {attachment.filename}."
+    response_message = f"Thanks for uploading, {interaction.user.display_name}! Received {language} file: {attachment.filename}."
 
     DIFFICULTIES = {"Easy": 1, "Medium": 2, "Hard": 3}
 
     if not submission.get("err"):
         difficulty_str = store["cotd"]["difficulty"]
-
-        # TODO: figure out who is first
-        was_first = False
-        # TODO: add up points from first accepted submission bonus!
         points = DIFFICULTIES[difficulty_str]
+        BONUS_POINTS = 1
+        was_first_submission = False
+
+        if store.__getitem__("first_submission") == False:
```

Comment:
This still stores the "true" value before confirming that the person's submission was accepted or not

Reply:
If I submit first and it isn't accepted, the first bonus should still be available.